### PR TITLE
G Suite account policy draft

### PR DESCRIPTION
**Pull request by 0x2b3bfa0**
This pull request adds a basic G Suite account policy stub. It probably needs to be to be rewritten from scratch and expanded to fit @slash-under criteria, but at least it mitigates the feared blank screen syndrome.

### CODE REVIEW

File: documentation/architecture/cloud/gsuite/accounts.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Helpful Engineering G Suite - Account Policy
+
+## Types
+Accounts can be divided in two major categories: personal accounts, external accounts and purpose-specific accounts. The latter should be avoided whenever possible lest losing fine-grained control over individual permissions, but they're required for some specific use cases like Google Analytics and common file storage.
```
Comment (imonthercks) [inline documentation/architecture/cloud/gsuite/accounts.md]:
```suggestion
Accounts can be divided in three major categories: personal accounts, external accounts and shared (purpose-specific) accounts. Shared accounts will be created only in very specific cases where several individuals need access to a single account for access to G Suite managed resources such as Google Analytics and common file storage.
```

Comment (0x2b3bfa0) [reply documentation/architecture/cloud/gsuite/accounts.md]: Nobody expects the Spanish inquisition. :smile: Thanks!

File: documentation/architecture/cloud/gsuite/accounts.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+#### Group mail
+In those cases where mailing is the only need for an «account», there is a much better alternative known as _shared inbox_, that allows multiple users to send and receive mail from their individual accounts without additional overhead. These shared inboxes can be managed through [Google Groups](https://groups.google.com/a/helpfulengineering.org/forum/#!myforums), nearly like normal account inboxes. 
+
+## Naming
+Though there were some exceptions in the early stages of the provisioning process, all the newly created personal accounts should be named following the well-known convention of concatenating the initial of the first name and the complete last name (*exempli gratia:* John Doe => `jdoe@helpfulengineering.org`). <!-- let's hope we don't have any user called Ajani Erkson -->
```
Comment (imonthercks) [inline documentation/architecture/cloud/gsuite/accounts.md]:
```suggestion
Though there were some exceptions in the early stages of the provisioning process, all the newly created personal accounts should be named following the well-known convention of concatenating the initial of the first name and the complete last name (*exempli gratia:* John Doe => `jdoe@helpfulengineering.org`). <!-- let's hope we don't have any user called Ajani Erkson -->  In the case where this convention produces duplication, additional letters from the first name should be used to generate a unique username: a second user with the name John Doe would result in an address of j**o**doe@helpfulengineering.org.
```

**Review by imonthercks — APPROVED** [review]
@0x2b3bfa0, I suggested a couple of changes, but they are both subjective changes, so I will approve and rely on any other comments to judge whether they are worthwhile.

**Comment by 0x2b3bfa0** [comment]
Thanks! They look good to me.

**Comment by 0x2b3bfa0** [comment]
Should we adapt and extend this document to fit [this template](https://github.com/helpfulengineering/devops/blob/master/documentation/architecture/review/template.md)? So, @slash-under might be able to document many other G Suite peculiarities.

**Comment by imonthercks** [comment]
@0x2b3bfa0 might be overkill for this doc

**Comment by 0x2b3bfa0** [comment]
Then we probably should move it to [guidance](https://github.com/helpfulengineering/devops/tree/master/documentation/guidance) to avoid mixing things.

File: documentation/architecture/cloud/gsuite/accounts.md [hunk 3]
```diff
@@ -16,7 +16,7 @@ This kind of account, managed by one or many users through a [securely shared pa
...
 
 ## Naming
-Though there were some exceptions in the early stages of the provisioning process, all the newly created personal accounts should be named following the well-known convention of concatenating the initial of the first name and the complete last name (*exempli gratia:* John Doe => `jdoe@helpfulengineering.org`). <!-- let's hope we don't have any user called Ajani Erkson -->  In the case where this convention produces duplication, additional letters from the first name should be used to generate a unique username: a second user with the name John Doe would result in an address of j**o**doe@helpfulengineering.org.
+Though there were some exceptions in the early stages of the provisioning process, all the newly created personal accounts should be named following the well-known convention of concatenating the initial of the first name and the complete last name (*exempli gratia:* John Doe => `jdoe@helpfulengineering.org`). In the case where this convention produces duplication, additional letters from the first name should be used to generate a unique username: a second user with the name John Doe would result in an address of `j**o**doe@helpfulengineering.org.
```
Comment (0x2b3bfa0) [inline documentation/architecture/cloud/gsuite/accounts.md]:
```suggestion
Though there were some exceptions in the early stages of the provisioning process, all the newly created personal accounts should be named following the well-known convention of concatenating the initial of the first name and the complete last name (*exempli gratia:* John Doe => `jdoe@helpfulengineering.org`). In the case where this convention produces duplication, additional letters from the first name should be used to generate a unique username: a second user with the name John Doe would result in an address of `jodoe@helpfulengineering.org`.
```

Those Spanish ISO dead keys...

**Review by dmangot — APPROVED** [review]
I agree, this is good enough to avoid blank page syndrome, and should probably be marked as DRAFT on the top.  I'd rather make PRs against the merged document then start nitpicking the starting document.  I don't think the 3 types are that clear and it's obviously missing the discussion of Vaults.  But it's good enough to get started.  A diagram may help as well.

**Comment by 0x2b3bfa0** [comment]
> I agree, this is good enough to avoid blank page syndrome, and should probably be marked as DRAFT on the top.

Do you mean to prepend `DRAFT` to the title additionally to the grey draft icon added by GitHub?

> I'd rather make PRs against the merged document than start nitpicking the starting document.

Agreed! I'll merge it to [guidance](/documentation/guidance), at least provisionally.

> I don't think the 3 types are that clear and it's obviously missing the discussion of Vaults. But it's good enough to get started. A diagram may help as well.

I was unsure about which was the place to discuss account sharing details and finally it was [moved](https://github.com/helpfulengineering/devops/pull/69/files#diff-06b60316758add1ccdbd70c560f13c2bR13) to the corresponding guidance document, but we might keep it here instead.

Diagrams are an excellent idea, and the account type-based classification is subject to change. It was only a way of getting started.

**Pull request merged**